index.bs

```diff
@@ -2150,6 +2211,9 @@ spec:css-syntax-3;
         </pre>
       </div>
 
+  1.  If the new credential type supports {{CredentialMediationRequirement/conditional}} [=user
+      mediation=], overload {{Credential/isConditionalMediationAvailable()}} and return `true`.
```

Comment:
```suggestion
      mediation=], override {{Credential/isConditionalMediationAvailable()}} and return `true`.
```

or maybe

```suggestion
      mediation=], define {{ExampleCredential/isConditionalMediationAvailable()}} to return `true`.
```

since WebIDL doesn't say what it would mean to override a static method.